libs/sdk-ui-kit/src/Header/tests/generateHeaderMenuItemsGroups.test.ts

```diff
@@ -187,9 +349,31 @@ describe("generateHeaderMenuItemsGroups", () => {
         ]);
     });
 
+    it("should not return data item with workspace in uri if backendSupportsDataItem is false, platform edition is not free and enableRenamingProjectToWorkspace is true", () => {
```

Comment:
Please add one more test case when `enableRenamingProjectToWorkspace` is `false`.

`it("should not return data item with project in uri if backendSupportsDataItem is false, platform edition is not free and enableRenamingProjectToWorkspace is false"`

Reply:
As the existing test can be considered for case when `enableRenamingProjectToWorkspace` is `false`, the comment is marked as resolved.